bindings/2.206/GeometryDash.bro

```diff
@@ -3233,7 +3233,7 @@ class EditorUI : cocos2d::CCLayer, FLAlertLayerProtocol, ColorSelectDelegate, GJ
 	TodoReturn getGroupCenter(cocos2d::CCArray*, bool) = win 0x11ed20;
 	TodoReturn getGroupInfo(GameObject*, cocos2d::CCArray*, int&, int&, int&);
 	cocos2d::CCPoint getLimitedPosition(cocos2d::CCPoint) = win 0x11c280;
-	CCMenuItemSpriteExtra* getModeBtn(char const*, int) = ios 0x247D9C;
```

Comment:
did you mean to remove this binding

Reply:
It conflicts with `cocos2d::CCSprite::createWithSpriteFileName`

Reply:
oh aight